protoc-gen-gogo/generator/helper.go

```diff
@@ -442,6 +442,6 @@ func (d *Descriptor) Path() string {
 
 func (g *Generator) useTypes() string {
```

Comment:
We should make sure that this only gets called, when we actually use types and we could make the same method for useTime.
Is more work really needed for other customtypes?